.circleci/config.yml

```diff
@@ -334,9 +370,16 @@ jobs:
           fl_build_app_objdet: "OFF"
           fl_build_app_lm: "OFF"
       - test_flashlight_inside_nvidia_docker:
-          fl_test_apex_dir: "flashlight/lib/test"
+          test_name: "fl_pkg_speech"
```

Comment:
What's the CI wait time like on this particular job (`ubuntu2004_cuda11_1_docker_flashlight_pkgs`)? If we can parallelize across pkgs, that's well worth it and should cut down on the test time significantly

It'll also make it much easier to reason about which things have broken if one can identify which pkgs they may have broken

Reply:
The CI time for `ubuntu2004_cuda11_1_docker_flashlight_pkgs` is _just_ over `ubuntu2004_cuda11_1_docker_flashlight_fl_core` ( 26m vs 25 m ).  I agree with the benefits, and actually my first pass on this had each pkg as a separate CI job, but I thought it was a bit cumbersome having so many. Given the very slight time increase, I think I'd still prefer it as is, but let me know if you feel strongly.

Reply:
If there's no time increase, that's just fine, and we can leave it as is.